src/Paket/Commands.fs

```diff
@@ -608,6 +611,9 @@ with
             | Endpoint(_) -> "API endpoint to push to (default: /api/v2/package)"
             | Endpoint_Legacy(_) -> "[obsolete]"
 
+            | Ignore_Conflicts -> "Ignore any HTTP409 (Conflict) errors and treat as success"
+            | Ignore_Conflicts_Legacy(_) -> "[obsolete]"
```

Comment:
Ignore_Conflicts_Legacy - what is this?

Reply:
i copied the others....wasn't sure it it was needed